packages/react-app/src/App.js

```
@@ -0,0 +1,69 @@
+                                navTimestamp: Date.now()
+                            });
+                        }
+                        else {
+                            navigateTo('logTransactionForm', { navTimestamp: Date.now() });
+                        }
+                    } });
+            case 'analyticsPage':
+                return _jsx(AnalyticsPage, {});
+            case 'accountsPage':
+                return _jsx(AccountsPage, {});
+            case 'dashboard':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)